pkg/cleanup/directories.go

```diff
@@ -45,19 +45,29 @@ func (d *directories) Run() error {
 	}
 
 	var dataDirMounted bool
+	allKubeletsUnmounted := true
```

Comment:
I'd like it if the logic was flipped so that the two bools work the same way.

So, something like:

```go
var kubeletDirsMounted bool
```

Then set it to true if unmount fails.

Reply:
the dataDirMounted variable only appears to control logging, not functional.